distribution/constellation64.bat

```diff
@@ -1,4 +1,5 @@
 @echo off 
 Title "Constellation"
 
+REG ADD "HKCU\SOFTWARE\Microsoft\Windows NT\CurrentVersion\AppCompatFlags\Layers" /v "%cd%\bin\constellation64.exe" /t REG_SZ /d "~ DPIUNAWARE" /f
```

Comment:
Is this registry addition compatible with other OS versions?

Reply:
Pretty sure this batch file is only included in the Windows distributions.